tools/editor/editor_settings.cpp

```diff
@@ -1022,6 +1022,23 @@ void EditorSettings::set_optimize_save(bool p_optimize) {
 	optimize_save=p_optimize;
 }
 
+String EditorSettings::get_last_selected_language()
+{
+	Ref<ConfigFile> cf = memnew( ConfigFile );
+	Error err = cf->load(get_project_settings_path().plus_file("project_metadata.cfg"));
+	if (err != OK) {
+		return "";
+	}
+	return static_cast<String>(cf->get_value("script_setup", "last_selected_language"));
```

Comment:
Maybe add `""` as default value to handle the parsing of an invalid config file: https://github.com/godotengine/godot/blob/master/core/io/config_file.cpp#L87